web/docs/tutorial/07-auth.md

```diff
@@ -271,33 +269,41 @@ wasp db studio
 ```
 
 <img alt="Database demonstration - password hashing"
-src={useBaseUrl('img/wasp_db_hash_demonstration.gif')}
+src={useBaseUrl('img/wasp_user_in_db.gif')}
 style={{ border: "1px solid black" }}
 />
 
-We see there is a user and that its password is already hashed 🤯
+
+You'll notice that we now have a `User` entity in the database alongside the `Task` entity.
 
 However, you will notice that if you try logging in as different users and creating some tasks, all users share the same tasks. That's because we haven't yet updated the queries and actions to have per-user tasks. Let's do that next.
 
+<small>
+
+You'll notice some extra entities like `Auth` and `AuthIdentity` that Wasp created for us. You don't need to care about these right now, but if you are curious, you can read more about them [here](/docs/auth/entities).
```

Comment:
```suggestion
You might notice some extra Prisma models like `Auth` and `AuthIdentity` that Wasp created for us. You don't need to care about these right now, but if you are curious, you can read more about them [here](/docs/auth/entities).
```

Reply:
I wrote this comment because these aren't entities really -> they are prisma models, but they are not "Wasp entities". Right? Or wait, are they, we did do some entities injecting in the generator hm? In that case disregard this, but you can still put "might".
Btw how would they notice these? Only if they inspected the DB, right?

Reply:
Yep, they'll notice them when inspecting the DB with Prisma studio :D